### DISCUSSION TITLE
What are the widely used Linux variants by Eclipse users for testing SWT features or fixes ?

### DISCUSSION THREAD

**Discussion by lathapatil** [eclipse-platform/eclipse.platform.swt#900] [Q&A] [answered]
What are the widely used Linux variants (Ubuntu , Fedora, Red Hat Enterprise Linux , openSUSE etc. ) by Eclipse users ? Which variants are considered for testing SWT features or bug fixes to ensure working of same on Linux OS ?

**Comment by vogella** [comment]
Just a tiny data point: I use Ubuntu

**Comment by mickaelistria** [comment, marked as answer]
Hi,
Fixes are always welcome as pull requests for any Linux distribution, as long as they don't cause trouble to others.
While https://www.eclipse.org/projects/project-plan.php?planurl=http://www.eclipse.org/eclipse/development/plans/eclipse_project_plan_4_29.xml#target_environments lists the environment considered as supported, there is no particular strategy which adds extra testing or resources targetting those distributions particularly; and the automated tests are currently only run on CentOS if I'm not mistaken; so the list is more a wish than a commitment.
Most popular distributions among Eclipse Platform developers are Fedora latest, Ubuntu latest and RHEL.

**Comment by laeubi** [comment]
Github action checks run on ubuntu for swt: https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources
